src/Outsystems.SetupTools/Functions/Install-OSServerPreReqs.ps1

```diff
@@ -257,8 +257,10 @@ function Install-OSServerPreReqs
             }
         }
 
+        # If IIS can't find ASP.NET modules install Hosting Bundle again to repair
+        $aspModules = Get-WebGlobalModule | where-object { $_.name -like "aspnetcoremodule*" }
```

Comment:
This should be inside the check phase of .NET Core.
https://github.com/OutSystems/OutSystems.SetupTools/blob/7fe004f2880ca86506b9fdae2f8edbe6a6ee2d9c/src/Outsystems.SetupTools/Functions/Install-OSServerPreReqs.ps1#L130-L142
Not here.
Change the "where-object" to "Where-Object", "name" to "Name"

Reply:
Added the test, updated CHANGELOG and changed "where-object" to "Where-Object".
I understand that the verification should be in the check phase of .NET Core but the powershell function Get-WebGlobalModule is only available after installing IIS.

Reply:
Yes, true. Didn't remember that. You can use something like this to check if the cmdLet is available:
if (Get-Command -Name "Get-WebGlobalModule" -ErrorAction SilentlyContinue) { <do your stuff> }
Also, if IIS is not installed doesnt make sense to check for the module.
And if we dont change the place where it is right now it will be applied to OS10.

Reply:
Reverted changes to Install-OSServerPreReqs.